stack/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ name: default-services
 services:
   postgres:
     container_name: postgres
-    image: postgres:16.3-alpine3.18@sha256:64e18e8fb3e9c9aac89ac590c5dd8306b862478404f76cd9b5f7720d012b4c47
```

Comment:
Did you look into switching this to `16.3-alpine`?

Reply:
Yes, but would stick with alpineX.X Images because we can bump alpine image updates more transparently.